DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -211,7 +211,7 @@ public void RegisterCentreManager(AdminRegistrationModel registrationModel, int
 
             CreateDelegateAccountForAdmin(registrationModel, jobGroupId);
 
-            registrationDataService.RegisterAdmin(registrationModel);
+            registrationDataService.RegisterAdmin(registrationModel, 0);

```

Comment:
Can we stick todos on these two lines? It's a bit scary if you come across a bit of code with weird hard-coded values and no explanation for it.